sdk/python/kfp/dsl/_container_op.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, name: str, image: str, command: str=None, arguments: str=None
     self.human_name = name
     self.name = _pipeline.Pipeline.get_default_pipeline().add_op(self, is_exit_handler)
     self.image = image
+    self.image_pull_policy = None
```

Comment:
Can we make the field private (leading underscore) and add a setter function? `set_image_pull_policy `